docs/service-support.adoc

```diff
@@ -85,9 +85,9 @@ The endpoint details shows an extended list of information about the app instanc
 |Status |Activity status of endpoint (activated /deactivated)
 |Type |Type of software as displayed today
 |Machine class |EFDI machine class, displayed for machines only
-|Onboarding Date & Time |Date and time when the endpoint was onboarded
+|Connection Date & Time |Date and time when the endpoint was onboarded
```

Comment:
Is it the real connection time? The AR says "Date and time of connection (UTC)" which means the date and time when the endpoint was initially connected (and I would suggest that as description).

Reply:
It's the time when the endpoint is created (Onboarded). Adjusted wording and added "UTC"